fabtests/pytest/sm2/conftest.py

```diff
@@ -1,11 +1,16 @@
 import pytest
 
-# TODO Restore GPU Memory Types
-@pytest.fixture(scope="module", params=["host_to_host"])
+@pytest.fixture(scope="module", params=["host_to_host",
+                                        pytest.param("host_to_cuda", marks=pytest.mark.cuda_memory),
+                                        pytest.param("cuda_to_host", marks=pytest.mark.cuda_memory),
+                                        pytest.param("cuda_to_cuda", marks=pytest.mark.cuda_memory),
+                                        pytest.param("neuron_to_neuron", marks=pytest.mark.neuron_memory),
+                                        pytest.param("neuron_to_host", marks=pytest.mark.neuron_memory),
+                                        pytest.param("host_to_neuron", marks=pytest.mark.neuron_memory)])
```

Comment:
All test stuff should be in a separate patch

Reply:
done